app/routes/circulars/circulars.server.ts

```diff
@@ -48,17 +49,29 @@ export async function search({
   query,
   page,
   limit,
+  advancedQuery,
 }: {
   query?: string
   page?: number
   limit?: number
+  advancedQuery?: {
+    startDate?: string | number
+    endDate?: string | number
+  }
```

Comment:
Move these out of the `advancedQuery` object:

```suggestion
  startDate?: string | number
  endDate?: string | number
```

Reply:
Made this change here and in relevant functions in [67335f3](https://github.com/nasa-gcn/gcn.nasa.gov/pull/1026/commits/67335f39365fe353b6c9a2d5b513efe572d1a73e). I think I fixed search being passed the default values (Infinity and -Infinity), but need to check if providing Infinity as a string to formatDateEpoch will return Infinity as a number